modules/checks.js

```diff
@@ -0,0 +1,88 @@
+const axios = require('axios');
+const RichEmbed = require('discord.js').RichEmbed;
+const checks = [
```

Comment:
With the objects you're defining here, It's much better if you use the following syntax...

For single line objects (notice the spaces inside the brackets):
```js
{ regex: /https?:\/\/hastebin\.com\/(\w+)(?:\.\w+)?/g, getLink: 'https://hastebin.com/raw/{code}' }
```

And for multi-line objects:
```js 
{
    checks: [/Establishing SSL connection without server's identity verification is not recommended\./],
    title: 'MySQL SSL Error',
    link: 'https://github\.com/lucko/LuckPerms/wiki/Storage-system-errors#mysql-ssl-errors'
}
```

Just makes it easier to read 👌